src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.html

```diff
@@ -6,105 +6,84 @@
       class="kuiViewContent"
     >
       <!-- Intro -->
-      <div class="kuiVerticalRhythm">
-        <div class="kuiBar">
-          <h1 class="kuiTitle">
-            Create index pattern
-          </h1>
-
-          <div class="kuiBarSection">
-            <!-- Include system indices -->
-            <label
-              ng-if="controller.isSystemIndicesCheckBoxVisible()"
-              class="kuiCheckBoxLabel"
-              for="indexPatternCreationIncludeSystemIndices"
+      <h1 class="euiTitle">
```

Comment:
Can we change lines 8-35 to:

```html
      <!-- Intro -->
      <h1 class="euiTitle euiTitle--medium">
        Create index pattern
      </h1>

      <div class="euiSpacer euiSpacer--small"></div>

      <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--justifyContentSpaceBetween euiFlexGroup--alignItemsEnd euiFlexGroup--responsive">
        <div class="euiFlexItem euiFlexItem--flexGrowZero">
          <div class="euiText">
            <p class="euiTextColor euiTextColor--subdued">
              Kibana uses index patterns to retrieve data from Elasticsearch indices for things like visualizations.
            </p>
          </div>
        </div>

        <div class="euiFlexItem euiFlexItem--flexGrowZero">
          <label
            ng-if="controller.isSystemIndicesCheckBoxVisible()"
            class="kuiCheckBoxLabel "
            for="indexPatternCreationIncludeSystemIndices"
          >
            <input
              type="checkbox"
              id="indexPatternCreationIncludeSystemIndices"
              class="kuiCheckBox"
              ng-model="controller.doesIncludeSystemIndices"
              ng-change="controller.onIncludeSystemIndicesChange()"
            >

            <span class="kuiCheckBoxLabel__text">
              Include system indices
            </span>
          </label>
        </div>
      </div>
```

This will render like this (a bit more space between the title and description):

![image](https://user-images.githubusercontent.com/1238659/34230393-d33c733a-e58c-11e7-8d2d-4df4980bd04f.png)